src/displayapp/screens/WatchFacePineTimeStyle.h

```diff
@@ -42,22 +42,19 @@ namespace Pinetime {
       private:
         uint8_t displayedHour = -1;
         uint8_t displayedMinute = -1;
```

Comment:
I would say it's better to use `UINT8_MAX` here instead of relying on underflow.

Reply:
There's no guideline about this, so it's more subjective and not really in the scope of this PR.